### PR TITLE
Add inband FEC and packet loss percentage getters and setters

### PR DESCRIPTION
**Pull request by nstafie**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -214,6 +218,42 @@ impl Encoder {
 		Ok(value)
 	}
 
+	/// Configures the encoder's use of inband forward error correction (FEC).
+	pub fn set_inband_fec(&mut self, value: bool) -> Result<()> {
+		let value = match value {
```
Comment (SpaceManiac) [inline src/lib.rs]: This *might* be more succinctly written like `let value = if value { 1 } else { 0 };`;

File: src/lib.rs [hunk 2]
```diff
@@ -214,6 +218,42 @@ impl Encoder {
...
+	/// Gets encoder's configured use of inband forward error correction.
+	pub fn get_inband_fec(&mut self) -> Result<bool> {
+		let mut value: i32 = 0;
+		let result = unsafe { ffi::opus_encoder_ctl(self.ptr, OPUS_GET_INBAND_FEC, &mut value) };
+		try!(check("opus_encoder_ctl(OPUS_GET_INBAND_FEC)", result));
+		let value = match value {
+			0 => false,
+			_ => true
+		};
+		Ok(value)
```
Comment (SpaceManiac) [inline src/lib.rs]: These five lines could be reduced to `Ok(value != 0)`.

File: src/lib.rs [hunk 3]
```diff
@@ -214,6 +218,42 @@ impl Encoder {
...
+	}
+
+	/// Sets the encoder's expected packet loss percentage.
+	pub fn set_packet_loss_perc(&mut self, value: i32) -> Result<()> {
+		let result = unsafe { ffi::opus_encoder_ctl(self.ptr, OPUS_SET_PACKET_LOSS_PERC, value) };
+		check("opus_encoder_ctl(OPUS_SET_PACKET_LOSS_PERC)", result)
```
Comment (SpaceManiac) [inline src/lib.rs]: Does opus return an error here if you pass it an out-of-range value? I would expect it to but it'd be nice if you could confirm.

File: src/lib.rs (hunk 2 above)
Comment (nstafie) [inline src/lib.rs]: Oh, yeah, how silly of me to do it like that...

File: src/lib.rs (hunk 3 above)
Comment (nstafie) [inline src/lib.rs]: Yeah, opus returns an "invalid argument" error with a code of -1, so `BadArg`.

File: src/lib.rs (hunk 1 above)
Comment (nstafie) [inline src/lib.rs]: I agree, much cleaner.

**Comment by nstafie** [comment]
Updated with feedback changes

**Pull request merged**